notebooks-cpd/Event_Store_Data_Analytics.ipynb

```diff
@@ -0,0 +1,664 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# IBM Db2 Event Store - Data Analytics using Python API \n",
+    "\n",
+    "IBM Db2 Event Store is a hybrid transactional/analytical processing (HTAP) system. It extends the Spark SQL interface to accelerate analytics queries. \n",
+    "\n",
+    "This notebook illustrates how the IBM Db2 Event Store can be integrated with multiple popular scientific tools to perform data analytics.\n",
+    "\n",
+    "***Pre-Req: Event_Store_Table_Creation***"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Connect to IBM Db2 Event Store\n",
+    "\n",
+    "Edit the values in the next cell"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CONNECTION_ENDPOINT=\"\"\n",
+    "\n",
+    "EVENT_USER_ID=\"\"\n",
+    "\n",
+    "EVENT_PASSWORD=\"\"\n",
+    "\n",
+    "# Port will be 1100 for version 1.1.2 or later (5555 for version 1.1.1)\n",
+    "PORT = \"30370\"\n",
+    "\n",
+    "DEPLOYMENT_ID=\"\"\n",
+    "\n",
+    "# Database name\n",
+    "DB_NAME = \"EVENTDB\"\n",
+    "\n",
+    "# Table name\n",
+    "TABLE_NAME = \"IOT_TEMPERATURE\"\n",
+    "\n",
+    "HOSTNAME=\"\"\n",
+    "\n",
+    "DEPLOYMENT_SPACE=\"\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "bearerToken=!echo `curl --silent -k -X GET https://{HOSTNAME}:443/v1/preauth/validateAuth -u admin:password |python -c \"import sys, json; print(json.load(sys.stdin)['accessToken'])\"`\n",
+    "bearerToken=bearerToken[0]\n",
+    "keystorePassword=!echo `curl -k --silent  GET -H \"authorization: Bearer {bearerToken}\" \"https://{HOSTNAME}:443/icp4data-databases/{DEPLOYMENT_ID}/zen/com/ibm/event/api/v1/oltp/keystore_password\"`\n",
+    "keystorePassword"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Import Python modules"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## Note: Only run this cell if your IBM Db2 Event Store is installed with IBM Cloud Pak for Data (CP4D)\n",
```

Comment:
try to always spell it out instead of using the acronym, but if you really need an acronym, I think we prefer CPD instead of CP4D now.